### PR TITLE
OCPBUGS#16038: Replace fake passwords with standard replace values

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-16038](https://issues.redhat.com/browse/OCPBUGS-16038)

Version(s):
4.14 through to 4.10

Link to docs preview:
* Changes touch on a lot of documentation. Here is a preview link to the [OCP Welcome page](https://62251--docspreview.netlify.app/openshift-enterprise/latest/welcome/index.html).
* To access the OCP Docs Guidelines preview, Click *Files changed* > Locate *doc_guidelines.adoc* > Click the three dots > Click *View file*. 

QE review:
- [x] QE approval not required. See Kathryn's comment. Changes are deemed internal to the docs' team. 

Additional information:
Reach out to me for more info. I can show you the InfoSec email and the source of the issue if required.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
I looked through the changes, and I think we can roll this up under a style change that doesn't need QE.

**Comment by tmalove** [comment]
Hi @dfitzmau I left a suggestion, but everything else looks good. There is not a preview for the update in the contributing guidelines doc (not sure if this is a requirement here if it can be generated).

/remove-label peer-review-in-progress
/label peer-review-done

File: modules/developer-cli-odo-creating-a-database-with-odo.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ This configuration ensures that when a database service is started, appropriate
 [source,yaml]
 ----
   databaseName: "sampledb"
-  databasePassword: "samplepwd"
+  databasePassword: "password"
   databaseUser: "sampleuser"
```
Comment (jeana-redhat) [inline modules/developer-cli-odo-creating-a-database-with-odo.adoc:53]: this one changes the pattern of the other values and is not in `<>`. Not sure what to suggest here, but maybe:
```suggestion
  databaseName: "<database_name>"
  databasePassword: "<password>"
  databaseUser: "<username>"
```

File: modules/metering-use-mysql-or-postgresql-for-hive.adoc [hunk 2]
```diff
@@ -81,8 +81,8 @@ spec:
         db:
           url: "jdbc:postgresql://postgresql.example.com:5432/hive_metastore"
           driver: "org.postgresql.Driver"
-          username: "REPLACEME"
-          password: "REPLACEME"
+          username: "username"
+          password: "password"
```
Comment (jeana-redhat) [inline modules/metering-use-mysql-or-postgresql-for-hive.adoc:85]:
```suggestion
          username: "<username>"
          password: "<password>"
```

File: modules/sbo-categories-of-exposable-binding-data.adoc [hunk 3]
```diff
@@ -69,8 +69,8 @@ kind: Secret
 metadata:
   name: hippo-pguser-hippo
 data:
-  password: "MTBz"
-  user: "Z3Vlc3Q="
+  password: "password"
+  user: "username"
```
Comment (jeana-redhat) [inline modules/sbo-categories-of-exposable-binding-data.adoc:73]:
```suggestion
  password: "<password>"
  user: "<username>"
```

File: modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc [hunk 4]
```diff
@@ -70,8 +70,8 @@ $ for i in username password host port type; do oc exec -it deploy/spring-petcli
 .Example output: With all the values from the secret resource
 [source,text]
 ----
-/bindings/spring-petclinic-pgcluster/username: hippo
-/bindings/spring-petclinic-pgcluster/password: KXKF{nAI,I-J6zLt:W+FKnze
+/bindings/spring-petclinic-pgcluster/username: username
+/bindings/spring-petclinic-pgcluster/password: password
```
Comment (jeana-redhat) [inline modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc:74]:
```suggestion
/bindings/spring-petclinic-pgcluster/username: <username>
/bindings/spring-petclinic-pgcluster/password: <password>
```

File: modules/sbo-methods-of-exposing-binding-data.adoc
Comment (jeana-redhat) [inline modules/sbo-methods-of-exposing-binding-data.adoc:1]: The ones in this file need `<>`

**Review by jeana-redhat — COMMENTED** [review]
Just a few that are not consistent still. Really, really appreciate you picking this change up!

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/developer-cli-odo-creating-a-database-with-odo.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/developer-cli-odo-creating-a-database-with-odo.adoc]: Updated. Yes, I'm unsure if quotes are needed or not, but I'll retain them to be safe.

File: modules/metering-use-mysql-or-postgresql-for-hive.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/metering-use-mysql-or-postgresql-for-hive.adoc]: Updated.

File: modules/sbo-categories-of-exposable-binding-data.adoc (hunk 3 above)
Comment (dfitzmau) [reply modules/sbo-categories-of-exposable-binding-data.adoc]: Updated.

File: modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc (hunk 4 above)
Comment (dfitzmau) [reply modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc]: Updated

File: modules/sbo-methods-of-exposing-binding-data.adoc
Comment (dfitzmau) [reply modules/sbo-methods-of-exposing-binding-data.adoc:1]: Updated

**Comment by dfitzmau** [comment]
Thanks for the review, @jeana-redhat . I updated the files accordingly. Please let me know your thoughts.

**Comment by jeana-redhat** [comment]
@dfitzmau 4.13+ working is not bad! will need to do 4.10-4.12 manually

**Pull request merged**